prow/spyglass/lenses/buildlog/buildlog.ts

```diff
@@ -163,12 +205,7 @@ async function handlePin(e: MouseEvent) {
   }
   const button = document.getElementById("annotate-pin");
   if (button) {
-    const firstEl = await highlightLines(artifact, start, end, 'focus-line', button.parentElement!.parentElement);
-    button.remove();
-    if (firstEl) {
-      clipLine(firstEl);
-      scrollTo(firstEl);
-    }
+    await focusLines(artifact, start, end, button.parentElement!.parentElement!.parentElement);
```

Comment:
That sure is a lot of `parentElement`!

At some point (perhaps in a future PR), it may be worth improving this. Two approaches come to mind:
- Add/use a class on the element you're looking for (so you can recursively ascend, which is slightly less structure dependent), or 
- Add a `data-` attribute to the pin so that it knows what line it's referring to, which you can then look up directly (removing all structure dependence).

Reply:
Done